tournaments/forms.py

```diff
@@ -0,0 +1,12 @@
+from django import forms
+from auth_app.models import Bot
+
+
+class TournamentForm(forms.Form):
+    tournament = forms.CharField(label='name')
+    bots_selected = forms.CharField(label='botsSelected', widget=forms.HiddenInput(), required=False)
+    bots = forms.ChoiceField(label='Bots', widget=forms.HiddenInput(), choices=[], required=False)
+
+    def setup_bots_choices(self):
+        bots = Bot.objects.all()
+        self.fields['bots'].choices = enumerate([bot.name for bot in bots])
```

Comment:
Why you select all Bots in DB? I think that you must use only online bots and you've already had 2 methods to do that in `bot_handler.py`.

1. get_users_data() # returns a list with online bots
2. get_online_bots(users) # returns a enumerate list of users: List.

Reply:
As we were talking, we should have a meet with the PM to define the values he want to show. I set by default all bot of database, anyway we can change it after... with a refactor